### ISSUE TITLE
Inaccurate input.position

### ISSUE THREAD

**Issue by ArtiomMat** [godotengine/godot#49807]
### Godot version

3.3.2

### System information

Linux, Ubuntu 20.04 LTS

### Issue description

The `input.position` argument from `_input` returns inaccurate results as `InputEventMouseMotion`.
When the cursor moves one pixel the input is registered, however the position isn't changed.
Only after the cursor moves about 3 pixels the value changes.

From what I know, not 100% sure on that, the Windows version works as it should.

### Steps to reproduce

1.
```gdscript
func _input(input):
	if input is InputEventMouseMotion:
		print_debug(input.position);
```
2.
Move the cursor and watch the debugger's output.

**Comment by Calinou** [comment]
Can you reproduce this if you call `Input.set_use_accumulated_input(false)` in the script's `_ready()` function?

**Comment by ArtiomMat** [comment]
> Can you reproduce this if you call `Input.set_use_accumulated_input(false)` in the script's `_ready()` function?

If by reproduce you mean it still happens then yes.

**Comment by ArtiomMat** [comment]
If you move the cursor slowly you will especially notice that there are many duplicate prints

**Comment by Calinou** [comment]
Can you reproduce this in Godot 3.2.3? If so, this issue may be a duplicate of https://github.com/godotengine/godot/issues/48369.

**Comment by Rubonnek** [comment]
This issue may also be a duplicate of #43492

**Comment by Rubonnek** [comment]
I'm not able to reproduce this issue under Arch Linux using the `3.x` branch at commit 56310caee232582bc83b0c682e814e9fed866e91. In fact I get more events whose position remain the same due to sub-pixel movements on very small movement deltas.
